src/improvements/tasks/eth/007-delayedweth-xfer/VALIDATION.md

```diff
@@ -0,0 +1,83 @@
+2. [Verifying the transaction input](#understanding-task-calldata)
+3. [Verifying the state changes](#state-changes)
+
+## Expected Domain and Message Hashes
+
+> [!CAUTION]
+> Before signing, ensure the below hashes match what is on your ledger.
+> ### Optimism Foundation Operations Safe
+  Domain Hash:     0x4e6a6554de0308f5ece8ff736beed8a1b876d16f5c27cac8e466d7de0c703890
+  Message Hash:    0xb8b13da617762a29a6ca91db63ff91252839fdf80838f90932031378427c965c
+
+## Understanding Task Calldata
+
+This document provides a detailed analysis of the final calldata executed on-chain for the OPCM upgrade to v2.0.0.
+
+By reconstructing the calldata, we can confirm that the execution precisely implements the approved upgrade plan with no unexpected modifications or side effects.
+
+# State Changes
+
+## Single Safe State Overrides and Changes
+
+This task is executed by the single Foundation Operations Safe. Refer to the
+[generic single Safe execution validation document](../../../../../SINGLE-VALIDATION.md)
+for the expected state overrides and changes.
+
+Additionally, Safe-related nonces [will increment by one](../../../../../SINGLE-VALIDATION.md#nonce-increments).
+
+## Other Nonces
+
+In addition to the Safe-related nonces mentioned [previously](#single-safe-state-overrides-and-changes), new contracts will also have a nonce value increment from 0 to 1.
+This is due to [EIP-161](https://eips.ethereum.org/EIPS/eip-161) which activated in 2016.
+
+This affects the newly deployed dispute games mentioned in ["State Diffs"](#state-diffs).
+
+## State Diffs
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state
+  changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain
+  Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+  ---
+  
+### `0x21429af66058bc3e4ae4a8f2ec4531aac433ecbc`  (DelayedWETH for PermissionedDisputeGame)
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000033`
+  - **Before:** `0x0000000000000000000000009ba6e03d8b90de867373db8cf1a58d2f7f006b3a`
+  - **After:** `0x0000000000000000000000005a0aae59d09fccbddb6c6cceb07b7279367c3d2a`
+  - **Summary:** Owner of the DelayedWETH contract for the PermissionedDisputeGame transferred to the ProxyAdmin owner.
+  - **Detail:** Address of the ProxyAdmin owner can be verified in the [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/b3d020de42abeebeb5786ea5508aa08d12bdf4cd/superchain/configs/mainnet/op.toml#L45).
```

Comment:
Normally we provide some info to prove that this is indeed the correct address for the Permissioned DelayedWeth. 

Unfortunately the registry is outdated, but [this PR](https://github.com/ethereum-optimism/superchain-registry/pull/992) updates it so that you could link to the FaultDisputeGame address then do `cast call 0x1Ae178eBFEECd51709432EA5f37845Da0414EdFe 'weth()(address)'.